docker/images/nginx/Dockerfile

```diff
@@ -1,3 +1,16 @@
 FROM nginx
+
+# Creating self signed certificate for localhost
+RUN mkdir /etc/nginx/ssl
+
+RUN openssl req \
+        -x509 \
+        -nodes \
+        -newkey rsa:2048 \
+        -keyout /etc/nginx/ssl/server.key \
+        -out /etc/nginx/ssl/server.crt \
+        -days 1000 \
+        -subj "/CN=127.0.0.1"
```

Comment:
I don't think we should include a private key in a  public docker image, even for 127.0.0.1. I believe a more correct approach, although less convenient, would be to document a version of this command (possibly wrapped in a "[make generate-cert](https://github.com/mdn/viewsourceconf/pull/98/files#diff-b67911656ef5d18c4ae36cb6741b7965R75)" or similar) to be run on the developer's laptop, and then [mount the cert and key](https://github.com/mdn/viewsourceconf/pull/98/files#diff-b67911656ef5d18c4ae36cb6741b7965R24) from the host at runtime.